components/automate-deployment/pkg/manifest/client/http.go

```diff
@@ -137,18 +159,29 @@ func NoVerify(noVerify bool) Opt {
 // GetCurrentManifest retrieves the current manifest for the given
 // channel.
 func (c *HTTP) GetCurrentManifest(ctx context.Context, channel string) (*manifest.A2, error) {
-	url := fmt.Sprintf(c.latestManifestURLFmt, channel)
-	return c.manifestFromURL(ctx, url)
+	//try to get semantic version manifest
+	url := fmt.Sprintf(c.latestSemanticManifestURLFmt, channel)
+	m, err := c.manifestFromURL(ctx, url, channel)
+	if err == nil {
+		return m, nil
+	}
+	if strings.Contains(err.Error(), "failed to locate manifest") {
+		//since received error in fetching semantic version, try to fetch timestamp versioned manifest
+		url = fmt.Sprintf(c.latestManifestURLFmt, channel)
+		return c.manifestFromURL(ctx, url, channel)
+	}
+
+	return nil, err
 }
 
 // GetCurrentManifest retrieves the current manifest for the given
 // channel.
 func (c *HTTP) GetManifest(ctx context.Context, release string) (*manifest.A2, error) {
 	url := fmt.Sprintf(c.manifestURLFmt, release)
-	return c.manifestFromURL(ctx, url)
+	return c.manifestFromURL(ctx, url, "current")
```

Comment:
@rickmarry  we should not hard code the channel, it should come from the deployment configuration.

Reply:
@Dmaddu fixed. thanks